src/test/java/uk/gov/hmcts/reform/sendletter/services/LetterServiceTest.java

```diff
@@ -120,4 +118,16 @@ public void should_rethrow_runtime_exception_if_invocation_fails() throws Except
         verify(queueClient).sendAsync(any(Message.class));
         verifyNoMoreInteractions(queueClient);
     }
+
+    @Test
+    public void should_not_allow_null_service_name() throws Exception {
+        assertThatThrownBy(() -> service.send(letter, null))
+            .isInstanceOf(IllegalStateException.class);
+    }
+
+    @Test
+    public void should_not_allow_empty_service_name() {
```

Comment:
This isn't strictly related with this pull request, but it would be worth having a test checking if the message we send contains the expected body.